engine/table/src/main/java/io/deephaven/engine/util/BigDecimalUtils.java

```diff
@@ -71,27 +73,37 @@ public static PrecisionAndScale computePrecisionAndScale(
         // We first use the initial PRECISION_SCALE_N_TO_INSPECT values to estimate the max(precision - scale) and
         // max(scale), which corresponds to max(digits left of the decimal point), max(digits right of the decimal
         // point). Then we convert to (precision, scale) before returning.
-        int maxPrecisionMinusScale = 0;
-        int maxScale = 0;
+        int maxPrecisionMinusScale = -1;
+        int maxScale = -1;
         try (final ChunkSource.GetContext context = source.makeGetContext(PRECISION_SCALE_N_TO_INSPECT);
```

Comment:
Rename PRECISION_SCALE_N_TO_INSPECT ?





[1:41](https://illumon.slack.com/archives/C04DJDAM0CT/p1670006500747539)
Now is more of a working buffer size than a limit